scripts/deno/peer-dependencies.ts

```diff
@@ -0,0 +1,105 @@
+import { octokit } from './github.ts';
+
+const peers = async () => {
+	// yarn install --check-files
+	const process = Deno.run({
+		cwd: './dotcom-rendering/',
+		cmd: ['make', 'reinstall'],
+		stdout: 'null',
+		stderr: 'piped',
+	});
+
+	const [{ code }, rawOutput] = await Promise.all([
+		process.status(),
+		process.stderrOutput(),
+	]);
+	if (code !== 0) Deno.exit(code);
+
+	return new TextDecoder()
+		.decode(rawOutput)
+		.split('\n')
+		.filter((line) => line.includes('has incorrect peer dependency'))
```

Comment:
this will break if yarn output changes (or this project gets a new package manager) – what about using something like https://www.npmjs.com/package/validate-peer-dependencies?

Reply:
Yes, that might be a good future improvement! I’m not massively interested in fixing straight away, as PNPM should make this unnecessary.

Reply:
I’ve actually tried this package, but could not get it to work at all!